content/en/blog/the_sandbox_launch.md

```diff
@@ -0,0 +1,54 @@
+---
+title : "The ODC Sandbox Launch"
+description: "OpenData Community Hackathon: Partners host educational Twitter Spaces"
+lead: "Fight Sybils, Decentralize Data"
+contributors: ["Stefi"]
+date: 2023-02-19T22:26:54+01:00
+lastmod: 2023-02-19T22:26:54+01:00
+draft: false
+images: []
+---
+
+Our last hackathon occurred from Jan 5th till Jan 31st , competitors collaborated to 4 different bounties, each with a different focus like helping the community growth, developing dashboards,
+identifying sybil behaviors through exploratory analysis and packaging algorithms in “Legos” that can be chained together with the intention to collaborate 
+for an accurate classification of sybil wallets.  Details about the competition can be found in this [post](https://opendatacommunity.org/blog/data-builder-hackathon/) about the hackathon opening.  
+
+We had an intense month with projects feedback sections together with FDD analysts, twitter spaces with sponsors and a lot more
+. What about the results? Incredible! We had 53 submissions and 29 winning projects that shared 40K in prizes, the projects range from python packages to 
+whole decentralized data frameworks that can integrate live dashboards, details and other results can be here on this [hackathon wrap up presentation](https://docs.google.com/presentation/d/1Ic4p9o2pLHETjBv1N9v4wkLTYToZxh-GP6Bo8ICtkko/edit#slide=id.g20c507cff6c_0_1)
+Most important of all, some projects developed in the hackathon demonstrate great value, scalability and potential to be strong weapons in the fight against sybil 
+attacks, mainly in the distribution of public funds.  
+
+Therefore, we are very pleased to announce the birth of the OpenData Community Sandbox, a space to foster the maturation of these projects.
+In the coming weeks, the community will discuss which will be the initial projects invited to join the Sandbox, 
+these projects will be able to rely on numerous benefits such as:
+
+**Tooling**
+- Repositories inside ODC github   
+- Dedicated Discord channel  
+- Educational help on setting tools up   
+  
+**Promotion**   
+- web site listings   
+- Twitter spaces   
+- Ongoing promotion through ODC channels   
+- Featured inclusion in the [ODC Landscape](https://opendatacommunity.org/docs/landscape/)
+- Featured inclusion in our data stack   
+
+**Technical help**
+- Mentorship   
+- Feature in Hackathons   
+- Ongoing hackathon - aka bounties to accelerate and attract contributions  
+
+**Network**  
+- Community builders engagement and contributions (including some FDD/Gitcoin members) 
+- Other DAOs core members   
+- Possible sponsors  
+
+To follow the development of the Sandbox keep an eye on our [ODC Sandbox github repo](https://github.com/OpenDataforWeb3/sandbox) and our channels, 
+till 1st of April we will already have projects set to start development and accept contributions. Join us!   
+
```

Comment:
edit to 22-49:

Therefore, we are very pleased to announce the birth of the OpenData Community Sandbox, a space to foster the maturation of these projects.
In the coming weeks, the community will discuss which projects will be the initial ones invited to join the Sandbox, these projects will be able to rely on numerous benefits such as:

**Tooling**
- Repositories inside ODC github   
- Dedicated Discord channel  
- Educational help on setting tools up   
  
**Exposure**   
- Web site listings   
- Twitter spaces   
- Ongoing promotion through ODC channels   
- Featured inclusion in the [ODC Landscape](https://opendatacommunity.org/docs/landscape/)
- Featured inclusion in our data stack   

**Technical help**
- Mentorship   
- Feature in Hackathons   
- Bounties to accelerate and attract contributions  

**Network**  
- People that have a passion for open data and an open internet
- Community builders and contributtors (including some FDD/Gitcoin members) 
- Various DAO core members and some prolific figures from web3
- Possible sponsors

To follow the development of the Sandbox keep an eye on our [ODC Sandbox github repo](https://github.com/OpenDataforWeb3/sandbox) and our channels. 
Our goal is to have projects set to start development and accept contributions until the 1st of April. Join us in building our open data dreams!

Reply:
adjusted in the new commit, thanks